main.tf

```diff
@@ -0,0 +1,6 @@
+module "tf_label" {
+  source    = "git::https://github.com/cloudposse/tf_label.git?ref=0.1.0"
```

Comment:
Can we use `git::https://github.com/cloudposse/tf_label?ref=0.1.0`? 
Without `.git`